akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -185,4 +185,12 @@ instead either run under `/system/localReceptionist` or `/system/clusterReceptio
 The path change makes it impossible to do a rolling upgrade from 2.5 to 2.6 if you use Akka Typed and the receptionist
 as the old and the new nodes receptionists will not be able to communicate.
 
- 
\ No newline at end of file
+
+### Akka Typed API changes
```

Comment:
there is a new commit on master adding an "Akka Typed" section. I'm moved "Akka Typed API changes" to be a sub-header of "Akka Typed"